resource.go

```diff
@@ -85,7 +85,7 @@ var resourceMappings = []resourceMap{
 	},
 }
 
-func DefaultMapResource(res *resource.Resource) *monitoredrespb.MonitoredResource {
+func defaultMapResource(res *resource.Resource) *monitoredrespb.MonitoredResource {
```

Comment:
@fabxc, to fix the golint error I have unexported it. If it was exported intentionally then I'll revert this and add comment to fix the lint error. Let me know if it was intentional.

Reply:
@fabxc if you don't mind can you please comment on this? thx.